Engine/Results/LiveTradingResultHandler.cs

```diff
@@ -474,32 +477,29 @@ private void SetNextStatusUpdate()
             // First add send charts
 
             // Loop through all the charts, add them to packets to be sent.
-            // Group three charts to a packets, and add in the data to the chart depending on the subscription.
-
+            // Group three charts per packet
             foreach (var deltaChart in deltaCharts.Values)
             {
-                var chart = new Chart(deltaChart.Name);
-                current.Add(deltaChart.Name, chart);
+                current.Add(deltaChart.Name, deltaChart);
 
-                if (deltaChart.Name == _subscription || (_subscription == "*" && deltaChart.Name == "Strategy Equity"))
-                {
-                    chart.Series = deltaChart.Series;
-                }
-
-                // If there is room left in the group. add the subscription
-                // to the packet unless it is a wildcard subscription
-                if (current.Count >= groupSize && _subscription != "*")
+                if (current.Count >= groupSize)
                 {
                     // Add the micro packet to transport.
                     chartPackets.Add(new LiveResultPacket(_job, new LiveResult { Charts = current }));
+
                     // Reset the carrier variable.
                     current = new Dictionary<string, Chart>();
+                    if (chartPackets.Count >= 4)
+                    {
+                        // stream a maximum number of charts: 3 * 4 = 12
```

Comment:
Magic numbers like this might look good as a config option. `GetInt("streamed-chart-limit", 12)`

Reply:
Great review 👍